internal/operators/lvm/config.go

```diff
@@ -12,13 +15,18 @@ type Config struct {
 }
 
 // count all disks of drive type ssd or hdd
-func (o *operator) getValidDiskCount(disks []*models.Disk, installationDiskID string) int64 {
+func (o *operator) getValidDiskCount(disks []*models.Disk, installationDiskID string) (int64, error) {
 	var countDisks int64
+	var err error
 
 	for _, disk := range disks {
 		if (disk.DriveType == models.DriveTypeSSD || disk.DriveType == models.DriveTypeHDD) && installationDiskID != disk.ID && disk.SizeBytes != 0 {
-			countDisks++
+			if disk.SizeBytes < conversions.GbToBytes(o.config.LvmMinDiskSizeGB) {
+				err = fmt.Errorf("ODF LVM requires all the non-bootable disks to be more than %d GB", o.config.LvmMinDiskSizeGB)
```

Comment:
What was the reasoning? before this PR 1 satisfying disk was enough right?

Reply:
The host should have atleast one non-installtion disk that can be used to provision storage. The LVM operator failure was not related to disks, It was due to yaml syntax error. As I confirmed from Storage team, we need to add thin pool configuration for LVM cluster. I copied the thin pool configuration to the LVM cluster CR, additionaly tabs were copied. Its resolved Now.

Reply:
But here we enforce `ODF LVM requires *all* the non-bootable disks to be more than %d GB`
What am I missing?

Reply:
awesome, thank you